### ISSUE TITLE
Make it possible to delete just 1 event in a repeated sequence.

### ISSUE THREAD

**Issue by enoch85** [owncloud/calendar#676]
As it is today you it´s not possible to delete just one event in a repeated sequence of events. 

I would prefer if there would be a choice to delete only that particular event and not the whole series, like Google does it:

![image](https://cloud.githubusercontent.com/assets/4511254/5825818/8594cb40-a0ee-11e4-968d-973e0094cbee.png)

**Comment by donlencho** [comment]
Would be great! Absolutely agree it's one of the samll things that would make OC calendar on par with other calendars.

**Comment by brantje** [comment]
+1 would love it. Just deleted a repeated event, and i lost all events prior and after that day -_-'.

**Comment by brantje** [comment]
Might be related: #618

**Comment by kiliankoe** [comment]
Agree, just had the same thing happen on accident. Owncloud does nothing to warn the user that he/she's deleting more than the intended event.

**Comment by Owncloudp** [comment]
Really need this feature for calendar collaboration.

**Comment by rruru** [comment]
I also think it would be an important and necessary modification. Otherwise , there are many compatibility issues with other calendars via CalDav .
However , the application is great.
A greeting and thanks .
